### PR TITLE
fix(facility): cancel button is malfunction in facility update page

### PR DESCRIPTION
**Pull request by cp-Coder**
# Bug Fix

## Proposed Changes

- Removed sumbission of facility form on `cancel` button

## Associated Issue

- Fixes https://github.com/coronasafe/care_fe/issues/4201

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug/test a new feature.
- [ ] Update product[documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care)
- [ ] Ensure that UI text is kept in I18n files
- [ ] Prep the screenshot or demo video for the changelog entry, and attach it to issue
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/FacilityCreate.tsx [hunk 1]
```diff
@@ -470,7 +469,7 @@ export const FacilityCreate = (props: FacilityProps) => {
           Notification.Success({
             msg: "Facility updated successfully",
           });
-          navigate(`/facility/${facilityId}`);
+          // navigate(`/facility/${facilityId}`);
```
Comment (khavinshankar) [inline src/Components/Facility/FacilityCreate.tsx]: commenting this line would stop it from redirecting to facility page on successful update, we will need this

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Components/Facility/FacilityCreate.tsx (hunk 1 above)
Comment (aeswibon) [reply src/Components/Facility/FacilityCreate.tsx]: Sorry, I was checking something and forgot to uncomment it.

File: src/Components/Facility/FacilityCreate.tsx [hunk 2]
```diff
@@ -984,7 +983,7 @@ export const FacilityCreate = (props: FacilityProps) => {
             >
               <button
                 className="btn btn-default border-2 border-grey-300"
-                onClick={() => goBack()}
+                onClick={() => navigate(`/facility/${facilityId}`)}
```
Comment (khavinshankar) [inline src/Components/Facility/FacilityCreate.tsx]: @cp-Coder can you try event.preventDefault(), that should fix this, if that doesn't work then let's try hard coding the navigation path

**Comment by nihal467** [comment]
@cp-Coder what is the status on the PR

**Comment by nihal467** [comment]
@cp-Coder 
image

when we click the update facility button, it is not getting redirected to facility details page

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**